pkg/storage/plan_collaboratorStore.go

```diff
@@ -106,37 +108,33 @@ func (s *Store) PlanCollaboratorUpdate(
 
 // PlanCollaboratorDelete deletes the plan collaborator for a given id
 func (s *Store) PlanCollaboratorDelete(
+	tx *sqlx.Tx,
```

Comment:
If changing the signature, I think this is good opportunity to update the method to match the new paradigm (package level function vs receiver method)